include/sparta/PowersetAbstractDomain.h

```diff
@@ -235,6 +235,9 @@ class PowersetAbstractDomain
       return this->get_value()->contains(e);
     }
     }
+    SPARTA_ASSERT(false && "unknown AbstractValueKind");
+    // Return false to suppress -Wreturn-type warning reported by gcc
+    return false;
```

Comment:
I'm curious to see if a different compiler that understands the switch case always returns would give a warning here mentioning that this is dead code
We will see what our CI says.

Reply:
clang 16 does not raise any warnings here with `-Wall -Wextra`, although I have not tried MSVC.